stack.go

```diff
@@ -71,13 +71,15 @@ var ErrNoFunc = errors.New("no call stack information")
 //    %s    source file
 //    %d    line number
 //    %n    function name
+//    %k    package name
```

Comment:
It is more accurate to say that `%k` prints the "last segment of the package path".